crawler/helpers.py

```diff
@@ -160,7 +156,7 @@ def map_lh_doc_to_sql_columns(doc) -> Dict[str, Any]:
             Dict[str, str] -- Dictionary of MySQL versions of fields
     """
     value = map_mongo_to_sql_common(doc)
-    dt = datetime.strftime(datetime.now(timezone.utc), MYSQL_DATETIME_FORMAT)
+    dt = datetime.now(timezone.utc)
```

Comment:
ah, so you don't need to format it for mysql at all, it just takes the datetime as is? neat